src/frontend/screens/Library/components/LibraryTopBar/index.tsx

```diff
@@ -0,0 +1,127 @@
+    text: string
+    defaultValue: boolean
+    onChange: (checked: boolean) => void;
+  }[],
+  isEpicLoggedin: string | undefined,
+  isGOGLoggedin: string | undefined
+}
+
+export function LibraryTopBar({
+  handleCategory,
+  category,
+  filters,
+  setSelectedFilter,
+  selectedFilter,
+  otherFiltersData,
+  isEpicLoggedin,
+  isGOGLoggedin
+}: LibraryTopBarInterface): JSX.Element {
+  const { t } = useTranslation()
+  return (<Tabs
+          onTabChange={(val: Category) => handleCategory(val)}
+          defaultValue={category}
+        >
+          <Tabs.List className={styles.tabsList} type="outline">
+            <Tabs.Tab value="all">
+              <div className="menu">{t('ALL', 'ALL')}</div>
+            </Tabs.Tab>
+            <Tabs.Tab value="hyperplay">
+              <div className="menu">{t('HyperPlay')}</div>
+            </Tabs.Tab>
+            {isEpicLoggedin && (
+              <Tabs.Tab value="legendary">
+                <div className="menu">EPIC</div>
+              </Tabs.Tab>
+            )}
+            {isGOGLoggedin && (
+              <Tabs.Tab value="gog">
+                <div className="menu">GOG</div>
+              </Tabs.Tab>
+            )}
+            <Tabs.Tab value="sideload">
+              <div className="menu">{t('Other')}</div>
+            </Tabs.Tab>
+            <div>
+              <Dropdown
+                options={filters}
+                onItemChange={setSelectedFilter}
+                selected={selectedFilter}
+                targetWidth={275}
+              />
+            </div>
+            <div>
+              <GenericDropdown
+                target={
+                  <GenericDropdown.GenericButton
+                    text={'Other filters'}
+                    style={{ width: '340px' }}
+                  ></GenericDropdown.GenericButton>
+                }
+              >
+                {otherFiltersData.map((val, index) => (
+                  <Menu.Item
+                    closeMenuOnClick={false}
+                    key={`toggleItem${index}`}
+                  >
+                    <Toggle
+                      defaultChecked={val.defaultValue}
+                      labelPosition="right"
+                      onChange={
+                        //eslint-disable-next-line
+                        (e: any) => {
+                          val.onChange(e.target.checked)
+                        }
+                      }
+                    >
+                      <div
+                        className="body"
+                        style={{
+                          paddingLeft: 'var(--space-sm)',
+                          margin: 'auto 0px'
+                        }}
+                      >
+                        {val.text}
+                      </div>
+                    </Toggle>
+                  </Menu.Item>
+                ))}
+              </GenericDropdown>
+            </div>
+            <div id="alignEnd">
+              {/* <div>
```

Comment:
We can remove this too

Reply:
we actually still need to implement this feature and these icons are set up perfectly. I'd rather just leave them https://www.figma.com/file/GKPELZxk551oVC3EDLgEDA/HP-Launcher?type=design&node-id=3508-25495&mode=design&t=JSbrhwRJhQ2LcoyW-4

Reply:
here's the issue: https://github.com/HyperPlay-Gaming/hyperplay-desktop-client/issues/93 

it's out of scope for this pr though